compiler/rustc_codegen_cranelift/src/intrinsics/mod.rs

```diff
@@ -203,7 +203,7 @@ pub(crate) fn codegen_intrinsic_call<'tcx>(
             sym::transmute => {
                 crate::base::codegen_panic(fx, "Transmuting to uninhabited type.", source_info);
             }
-            _ => unimplemented!("unsupported instrinsic {}", intrinsic),
+            _ => unimplemented!("unsupported intrinsics {}", intrinsic),
```

Comment:
Plural incorrect here, this is a single intrinsic

Reply:
Already corrected, thank you.